pdc_client/runner.py

```diff
@@ -69,15 +69,26 @@ def __init__(self):
     def load_plugins(self):
         config = None
         server = None
+        idx_s, idx_server = (None, None)
         plugins_set = set(DEFAULT_PLUGINS)
         args = sys.argv[1:]
         try:
-            idx = args.index("-s")
-            server = args[idx + 1]
+            idx_s = args.index('-s')
         except ValueError:
             pass
         except IndexError:
             pass
+        try:
+            idx_server = args.index('--server')
+        except ValueError:
+            pass
+        except IndexError:
+            pass
```

Comment:
This kind of except could write something like:
```
except (ValueError, IndexError) as e:
    pass
```
Same as previous try....except......